.travis.yml

```
@@ -104,7 +104,7 @@ matrix:
       script:
         # This is a trivial test where we only check whether it compiles at all with all warnings enabled.
         # TODO: Write unit tests and run them on an emulator.
-        - avr-gcc libcanard/*.c -c -std=c11 -mmcu=at90can64 -Wall -Wextra -Werror -pedantic -Wconversion -Wtype-limits
+        - avr-gcc libcanard/*.c -c -std=c99 -mmcu=at90can64 -Wall -Wextra -Werror -pedantic -Wconversion -Wtype-limits
```

Do not remove existing tests. If you want to test something else, just add a new test nearby.

```suggestion
        - avr-gcc libcanard/*.c -c -std=c99 -mmcu=at90can64 -Wall -Wextra -Werror -pedantic -Wconversion -Wtype-limits
        - avr-gcc libcanard/*.c -c -std=c11 -mmcu=at90can64 -Wall -Wextra -Werror -pedantic -Wconversion -Wtype-limits
```